main.go

```diff
@@ -31,6 +32,7 @@ import (
 	fwebhook "github.com/fairwindsops/polaris/pkg/webhook"
 	"github.com/sirupsen/logrus"
 	apitypes "k8s.io/apimachinery/pkg/types"
+	yaml2 "k8s.io/apimachinery/pkg/util/yaml"
```

Comment:
Nitpick: Can this be named more appropriately? It's not `yaml` version 2 or something, it's specifically `apiMachineryYAML`. I think it would be beneficial to future readers to understand why there is a `yaml` and a `yaml2`. Non-blocking comment.